[1.7.9] Komodo server doesn't start

## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Documentation issue or request
</code></pre>

## Description
Because this property should be true/false, but it is not set during the template generation:

https://github.com/syndesisio/syndesis/blob/308b4cffd29cf85cb3276bec39f8ed775bce1efb/install/syndesis.yml#L2105-L2106

that results in this error in komodo-server:

```
***************************
APPLICATION FAILED TO START
***************************
Description:
Binding to target org.komodo.rest.KomodoConfigurationProperties@4466af20 failed:
    Property: controllers.exposeVia3scale
    Value: 
    Reason: Failed to convert property value of type 'java.lang.String' to required type 'boolean' for property 'exposeVia3scale'; nested exception is org.springframework.core.convert.ConverterNotFoundException: No converter found capable of converting from type [java.lang.String] to type [boolean]

```

@rkorytkowski 

```
[ERROR] createAndGetIntegration(io.syndesis.server.runtime.IntegrationsITCase)  Time elapsed: 4.376 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'integrationSupportHandler' defined in URL [jar:file:/workspace/app/server/endpoint/target/server-endpoint-1.8-SNAPSHOT.jar!/io/syndesis/server/endpoint/v1/handler/integration/support/IntegrationSupportHandler.class]: Unsatisfied dependency expressed through constructor parameter 5; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'integrationHandler': Unsatisfied dependency expressed through method 'setPublicApiHandler' parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'publicApiHandler': Unsatisfied dependency expressed through method 'setIntegrationSupportHandler' parameter 0; nested exception is org.springframework.beans.factory.BeanCurrentlyInCreationException: Error creating bean with name 'integrationSupportHandler': Requested bean is currently in creation: Is there an unresolvable circular reference?
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'integrationHandler': Unsatisfied dependency expressed through method 'setPublicApiHandler' parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'publicApiHandler': Unsatisfied dependency expressed through method 'setIntegrationSupportHandler' parameter 0; nested exception is org.springframework.beans.factory.BeanCurrentlyInCreationException: Error creating bean with name 'integrationSupportHandler': Requested bean is currently in creation: Is there an unresolvable circular reference?
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'publicApiHandler': Unsatisfied dependency expressed through method 'setIntegrationSupportHandler' parameter 0; nested exception is org.springframework.beans.factory.BeanCurrentlyInCreationException: Error creating bean with name 'integrationSupportHandler': Requested bean is currently in creation: Is there an unresolvable circular reference?
Caused by: org.springframework.beans.factory.BeanCurrentlyInCreationException: Error creating bean with name 'integrationSupportHandler': Requested bean is currently in creation: Is there an unresolvable circular reference?
```

--- a/install/generator/04-teiid-komodo-server.yml.mustache
+++ b/install/generator/04-teiid-komodo-server.yml.mustache
@@ -70,7 +70,7 @@
           - name: POSTGRESQL_DATABASE
             value: ${POSTGRESQL_DATABASE}
           - name: CONTROLLERS_EXPOSE_VIA3SCALE
-            value: '{{CONTROLLERS_EXPOSE_VIA3SCALE}}'            
+            value: ${CONTROLLERS_EXPOSE_VIA3SCALE}
 {{#Debug}}
           - name: JAVA_DEBUG
             value: "true"

--- a/install/syndesis.yml
+++ b/install/syndesis.yml
@@ -2097,7 +2097,7 @@ objects:
           - name: POSTGRESQL_DATABASE
             value: ${POSTGRESQL_DATABASE}
           - name: CONTROLLERS_EXPOSE_VIA3SCALE
-            value: ''            
+            value: ${CONTROLLERS_EXPOSE_VIA3SCALE}
           image: ' '
 
           imagePullPolicy: IfNotPresent

